### ISSUE TITLE
Column name handling: start/end

### ISSUE THREAD

**Issue by bschilder** [neurogenomics/MungeSumstats#96]
Currently, we don't have start/end in the mapping file. This isn't a problem for 1bp-wide features like SNPs (in the strict sense of the term), but if you include indels/structural variants you can both start/end columns spanning some range. 

Might we want to come up with some way to rename these such that it's compatible with the full pipeline? i.e. we probably want to keep "start" as "POS" since this is a cornerstone of how MSS works. But perhaps when synonyms of "end" occur, we can rename that something like "POS2".

**Comment by Al-Murphy** [comment]
This makes sense, we should probably only do it if the Indel parameter is set to true? Next time one of us is making changes to the dev branch it will probably be worth testing the effect of this. Do you kknow of any downstream software that uses start & end? What names do they require? Just want to make sure BP2 will be understandable to users and downstream applications

**Comment by Al-Murphy** [comment]
Yep happy to add the extra CHR mappings but put them in as upper case as all inputted headers are pushed toupper() anwyay (this doesn't really matter since the same is done to the entries in the mapping file but just so they are the same as what's there currently)

**Comment by Al-Murphy** [comment]
I actually had a few other concerns around checks on this with the reference datasets. Probably worth discussing in person before we commit to adding the functionality (and maybe including Nathan too)

**Comment by ricardovialle** [comment]
> SVs are a bit more complicated, but you can run GWAS with them very similar to the way you would run one with SNPs only. My former labmate did this in [AD](https://www.nature.com/articles/s41593-022-01031-7) so if we eventually decide to go in that direction it would be great to get his input. @ricardovialle would you mind giving some initial thoughts on this?

Hi guys, this definitely would be something nice to have. I'm not aware of a specific standard for reporting SVs summary stats. Even the VCFs created by SV callers sometimes do not follow the same specification. The [VCF format specification](https://samtools.github.io/hts-specs/VCFv4.3.pdf) usually expect an END field. In our results, we reported **pos** and **sv_end** ([link](https://github.com/RajLabMSSM/AMP_AD_StructuralVariation/tree/main/data/disease_associations)). Also, you might want to consider CIPOS and CIEND as many SVs have imprecise breakpoints.

**Comment by bschilder** [comment]
Thanks so much for the input @ricardovialle! Alan and I will chat about all this in our next meeting and we'll let you know the plan here.

**Comment by Al-Murphy** [comment]
Added extra CHR mappings
